doc/api/process.md

```diff
@@ -1939,7 +1939,17 @@ added: v0.5.9
 
 * `message` {Object}
 * `sendHandle` {net.Server|net.Socket}
-* `options` {Object}
+<<<<<<< Updated upstream
```

Comment:
Looks like you have a merge conflict to resolve.

Reply:
Thanks @Trott , I'll fix it,instead of this everything is ok or not.